boards/norik/octopus_io_board/octopus_io_board_common.dtsi

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2024 Norik Systems
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include "octopus_io_board_common-pinctrl.dtsi"
+#include "../octopus_som/octopus_som_common.dtsi"
+
+/ {
+	model = "Norik Octopus IO-Board";
+	compatible = "norik,octopus-io-board";
+
+	chosen {
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+		zephyr,uart-mcumgr = &uart0;
+	};
+
+	peripheral-power-controller {
+		compatible = "gpio-keys";
+
+		pwrgpio0: pwrgpio_0 {
+			label = "VP on/off";
+			gpios = <&gpio0 18 GPIO_ACTIVE_HIGH>;
+		};
+
+		pwrgpio1: pwrgpio_1 {
+			label = "CHG control";
+			gpios = <&gpio0 20 (GPIO_ACTIVE_LOW)>;
+		};
+
+	};
```

Comment:
these are not buttons, a custom dts is needed

Reply:
Fixed in recent changes